build/upgrade_pdfjs.sh

```diff
@@ -42,24 +42,19 @@ rm -rf ./pdfjs-dist/
 echo "-----------------------------------------------------------------------------------"
 echo "Decreasing # of cached pages on mobile web..."
 echo "-----------------------------------------------------------------------------------"
-sed -e 's@var DEFAULT_CACHE_SIZE = 10;@var DEFAULT_CACHE_SIZE = /iphone|ipad|ipod|android|blackberry|bb10|mini|windows\sce|palm/i.test(navigator.userAgent) ? 5 : 10;@' -i '' ${DOC_STATIC_ASSETS_PATH}/pdf_viewer.js
+sed -e 's@const DEFAULT_CACHE_SIZE = 10;@const DEFAULT_CACHE_SIZE = /iphone|ipad|ipod|android|blackberry|bb10|mini|windows\sce|palm/i.test(navigator.userAgent) ? 5 : 10;@' -i '' ${DOC_STATIC_ASSETS_PATH}/pdf_viewer.js
 
 # Render e-signatures without validation
 echo "-----------------------------------------------------------------------------------"
 echo "Enabling e-signature rendering without validation..."
 echo "-----------------------------------------------------------------------------------"
 sed -e 's@;r.setFlags(o.AnnotationFlag.HIDDEN)@@' -i '' ${DOC_STATIC_ASSETS_PATH}/pdf.worker.min.js
```

Comment:
I'm not sure this still works on the latest version. We should test digital signature to make sure they display properly.

Reply:
@karelee7, did we validate that this fix is still applied correctly and that digital signature still work?

Reply:
Yeah, tested rendering a digital signature and it looks fine!